test/ComposerGpgVerifyTest/VerifyTest.php

```diff
@@ -0,0 +1,445 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ComposerGpgVerifyTest;
+
+use Composer\Composer;
+use Composer\Config;
+use Composer\Script\Event;
+use Composer\Script\ScriptEvents;
+use ComposerGpgVerify\Verify;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Process\Process;
+
+/**
+ * @covers \ComposerGpgVerify\Verify
+ */
+final class VerifyTest extends TestCase
```

Comment:
Please add tests for the following scenarios:

- A repository which contains a mixture of good and bad signatures
-- If most recent is trusted/verified accept it
-- If most recent isn't trusted reject it.

I'm mostly considering a scenario where an attacker maliciously forks a package and adds bad commits/tags on top of good ones. Code looks like it should reject: test proves it does.